packages/design-system-react/src/components/button-primary/README.mdx

```diff
@@ -0,0 +1,77 @@
+import { Controls, Canvas } from '@storybook/blocks';
+
+import * as ButtonPrimaryStories from './ButtonPrimary.stories';
+
+# ButtonPrimary
+
+ButtonPrimary is used for primary actions
+
+```tsx
+import { ButtonPrimary } from '@metamask/design-system-react';
+
+<ButtonPrimary>Primary Button</ButtonPrimary>;
+```
+
+<Canvas of={ButtonPrimaryStories.Default} />
+
+## Props
+
+### Size
+
+ButtonPrimary supports three sizes:
+
+- `ButtonPrimarySize.Sm` (32px)
+- `ButtonPrimarySize.Md` (40px) - default
+- `ButtonPrimarySize.Lg` (48px)
+
+<Canvas of={ButtonPrimaryStories.Size} />
+
+### IsFullWidth
+
+ButtonPrimary can be set to take up the full width of its container.
+
+<Canvas of={ButtonPrimaryStories.IsFullWidth} />
+
+### IsDanger
+
+Use the danger variant for destructive actions.
+
+<Canvas of={ButtonPrimaryStories.IsDanger} />
+
+### IsInverse
+
+Use the inverse variant when the button needs to be displayed on a dark background. When combined with `isDanger`, the button will have inverse styling with error text color.
+
+<Canvas of={ButtonPrimaryStories.IsInverse} />
```

Comment:
Focus on these additional prop stories the reset should be the same as `ButtonBase` with update names